package.json

```diff
@@ -32,6 +32,8 @@
     "morgan": "^1.9.1",
     "multer": "^1.4.2",
     "pg": "^7.14.0",
+    "request": "^2.88.0",
+    "request-promise-native": "^1.0.8",
```

Comment:
Have you considered axios instead? 4kb vs request-promise's 66kb
see here: https://www.npmtrends.com/axios-vs-request-promise-vs-superagent

Reply:
I can look into it, but request-promise has a really handy feature I'm relying on to get not just the photo from Google, but THEIR hosted version of it. I don't want to change over to axios if it does not support a similar type of option (places-util#37)